packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -510,6 +511,122 @@ export class SingleValueWidget extends ConcreteWidget {
   }
 }
 
+/**
+ * The number of top contributors to show.
+ */
+export enum TopContributors {
+  /**
+   * The top 10 contributors
+   */
+  TOP10 = 10,
+  /**
+   * The top 25 contributors
+   */
+  TOP25 = 25,
+  /**
+   * The top 50 contributors
+   */
+  TOP50 = 50,
+  /**
+   * The top 100 contributors
+   */
+  TOP100 = 100,
```

Comment:
Generally we have underscores separating words in our enums. So I think we should have these be `TOP_100` and `TOP_50` and so on.